filebeat/docs/getting-started.asciidoc

```diff
@@ -56,20 +56,27 @@ include::{libbeat-dir}/tab-widgets/install-widget.asciidoc[]
 include::{libbeat-dir}/shared/connecting-to-es.asciidoc[]
 
 [float]
-[[enable-modules]]
-=== Step 3: Enable and configure data collection modules
+[[collect-log-data]]
+=== Step 3: Collect log data
+
+There are three ways to collect log data with {beatname_uc}:
```

Comment:
```suggestion
There are several ways to collect log data with {beatname_uc}:
```

Reply:
It's generally best to avoid introducing a list with a number. It's just easier to maintain.